stable/kibana/values.yaml

```diff
@@ -90,3 +100,177 @@ nodeSelector: {}
 
 podAnnotations: {}
 replicaCount: 1
+revisionHistoryLimit: 3
+
+# to export a dashboard from a running kibana 6.3.x use:
+# curl --user <username>:<password> -XGET https://kibana.yourdomain.com:5601/api/kibana/dashboards/export?dashboard=<some-dashboard-uuid> > my-dashboard.json
+# the commented out json is an example Kubernetes dasboard which works out of the box if you use the fluentd-elasticsearch chart to collect logs from your Kubernetes nodes
+ConfigmapDashboardImport:
+  xpackauth:
+    enabled: false
+    username: myuser
+    password: mypass
+  dashboards: {}
```

Comment:
Is there a better way than passing the full contents of each json file in here? Would config files referenced from the values file work?

Reply:
I have to try...

Reply:
I have to try...

Reply:
You cannot reference external files. I guess there's no better way.